CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 > - :house: [Internal]
 > - :nail_care: [Polish]
 
+#### :boom: Breaking Change
```

Comment:
Why is this a breaking change? This was never officially released right?

Reply:
You're correct. I'll fix it.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/5706/commits/628a5b172368d2755e1b4ae32266ce99c7cb740f

I just put it below `# master`.